src/get-user-trace.js

```diff
@@ -0,0 +1,21 @@
+import chalk from 'chalk'
+
+// Frame has the form "at myMethod (location/to/my/file.js:10:2)"
+function getFrameLocation(frame) {
+  const locationStart = frame.indexOf('(') + 1
+  const locationEnd = frame.indexOf(')')
+
+  return frame.slice(locationStart, locationEnd)
+}
+
+function getUserTrace() {
+  const err = new Error()
+  const firstClientCodeFrame = err.stack
+    .split('\n')
+    .slice(1) // Remove first line which has the form "Error: TypeError"
+    .find(frame => !frame.includes('node_modules/')) // Ignore frames from 3rd party libraries
+
+  return chalk.gray(`${getFrameLocation(firstClientCodeFrame)}\n`)
```

Comment:
How does this look like in a browser? If I understand https://github.com/chalk/chalk/issues/1 correctly then `chalk` doesn't work in a browser. Would be nice to have confirmation that `@testing-library/dom` still works outside of `node`.

Reply:
These changes are now working in the browser as well since we're loading modules conditionally if we're in node :partying_face: